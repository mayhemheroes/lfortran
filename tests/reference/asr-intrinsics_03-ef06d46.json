{
    "basename": "asr-intrinsics_03-ef06d46",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/intrinsics_03.f90",
    "infile_hash": "eda277452cad2d7bc3ef98963677fc276c09ce4d517756982a0d921d",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-intrinsics_03-ef06d46.stdout",
    "stdout_hash": "738b7ae4d5cbd362628220adcf323642ec4d646d2e15ccfaea8d0cbf",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}