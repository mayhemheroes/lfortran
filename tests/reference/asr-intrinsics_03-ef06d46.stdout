(TranslationUnit (SymbolTable 1 {intrinsics_03: (Program (SymbolTable 2 {cos: (ExternalSymbol 2 cos 4 cos lfortran_intrinsic_math [] cos Private), cos@scos: (ExternalSymbol 2 cos@scos 4 scos lfortran_intrinsic_math [] scos Private), x: (Variable 2 x Local () () Default (Real 4 []) Source Public Required .false.)}) intrinsics_03 [] [(= (Var 2 x) (FunctionCall 2 cos@scos 2 cos [((RealConstant 9.500000 (Real 4 [])))] (Real 4 []) (RealConstant -0.997172 (Real 4 [])) ()) ()) (Print () [(Var 2 x)] () ())]), iso_c_binding: (IntrinsicModule lfortran_intrinsic_iso_c_binding), iso_fortran_env: (IntrinsicModule lfortran_intrinsic_iso_fortran_env), lfortran_intrinsic_builtin: (IntrinsicModule lfortran_intrinsic_builtin), lfortran_intrinsic_math: (IntrinsicModule lfortran_intrinsic_math)}) [])
